downloader/downloader.py

```diff
@@ -7,13 +7,50 @@
 
 # Reference energies computed with Psi4 1.5 wB97M-D3BJ/def2-TZVPPD.
 
-atom_energy = {('Br', -1): -2574.2451510945853, ('Br', 0): -2574.1167240829964, ('C', -1): -37.91424135791358, ('C', 0): -37.87264507233593,
-               ('C', 1): -37.45349214963933, ('Ca', 2): -676.9528465198214, ('Cl', -1): -460.3350243496703, ('Cl', 0): -460.1988762285739,
-               ('F', -1): -99.91298732343974, ('F', 0): -99.78611622985483, ('H', 0): -0.4987605100487341, ('I', -1): -297.8813829975981,
-               ('I', 0): -297.76228914445625, ('K', 1): -599.8025677513111, ('Li', 1): -7.285254714046546, ('Mg', 2): -199.2688420040449,
-               ('N', -1): -54.602291095426494, ('N', 0): -54.62327513368922, ('N', 1): -54.08594142587869, ('Na', 1): -162.11366478783253,
-               ('O', -1): -75.17101657391741, ('O', 0): -75.11317840410095, ('O', 1): -74.60241514396725, ('P', 0): -341.3059197024934,
-               ('P', 1): -340.9258392474849, ('S', -1): -398.2405387031612, ('S', 0): -398.1599636677874, ('S', 1): -397.7746615977658}
+atom_energy = {'Br': {-1: -2574.2451510945853, 0: -2574.1167240829964},
```

Comment:
It would make sense to include a script to compute these energies. So, in the future, it is easy to add more elements.

Reply:
Good idea.  I checked it in.